### ISSUE TITLE
Auto Width feature?

### ISSUE THREAD

**Issue by nickcartery** [Spider149/own-carousel#3]
Does this repos have Auto Width feature like OwnCarouse Jquery version?

**Comment by Spider149** [comment]
no, it will automatically set the width of each element equally depending on the setting

**Comment by nickcartery** [comment]
Vỡi, người Việt à? :v Thêm tính năng Auto Width đi bro ơi, cái đấy hưu ích lắm, Glide JS cũng k có cái đấy

**Comment by Spider149** [comment]
e người Việt :)) này e làm cũng lâu rồi, dạo này e cũng hơi bận :)) a có thể đọc code và custom lại cái biến step xem, cái step đó là số px để translate, do e làm cái này là mặc định tất cả phần tử có cùng width nên sẽ calculate dựa vào width của cái carousel, a có thể custom lại cách tính toán của nó dựa trên cái index xem.
